rust-src/concordium_base/src/protocol_level_tokens/token_module_initialization_parameters.rs

```diff
@@ -0,0 +1,73 @@
+use concordium_base_derive::{CborDeserialize, CborSerialize};
+
+use super::{MetadataUrl, TokenAmount};
+use crate::protocol_level_tokens::token_holder::CborTokenHolder;
+
+/// These parameters are passed to the token module to initialize the token.
+/// The token initialization update will also include the ticker symbol,
+/// number of decimals, and a reference to the token module implementation.
+#[derive(Debug, Clone, PartialEq, CborSerialize, CborDeserialize)]
+pub struct TokenModuleInitializationParameters {
+    /// The name of the token
+    pub name:               String,
+    // /// A URL pointing to the token metadata
```

Comment:
Just wonder if we should mention that it includes an optional hash. Something along the line:
```suggestion
    /// A URL pointing to the token metadata with an optional `checksum` hash.
```